src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Square extends FigureTwo implements AreaOfFigure,InfoFigure {
+    //квадрат
+
+    public Square(String color, double firsSide) {
+        super(color, firsSide, firsSide);
+    }
+
+    @Override
+    public double detAreaOfFigure() {
+        return getSecondSide() * getFirsSide();
+    }
+
+    @Override
+    public String infoFigure() {
+        return ("Длина : " + getFirsSide() + " см"
+                + " Ширина : " + getSecondSide() + " см"
+                + " Площадь квадрата = " + detAreaOfFigure());
```

Comment:
Avoid using non-English text in your code. The checklist advises using only English in messages and code to maintain universality and consistency.